docs/router.md

```diff
@@ -0,0 +1,204 @@
+# About Routers
+
+`gpii.express.router` is a subclass of [`gpii.express.middleware`](middleware.md) that wraps
+[the  Express `router` object](http://expressjs.com/en/4x/api.html#router), which is simply a container for a collection
```

Comment:
Yes, as we established elsewhere, a "router" can also perform switching based on path and method. It's rather a shame that we're limited to express' primitives here - for example, the contentAwareMiddleware is only capable of switching to a collection of "handlers" rather than acting as a full-scale router (I'm assuming its former name of contentAwareRouter was simply a misnomer). This is fine, but it might be cool to establish something similar to Kettle's idea of "per-handler middleware", thus closing up one more point of difference between them